src/components/ConversationSettings/DangerZone.vue

```diff
@@ -4,78 +4,109 @@
 -->
 
 <template>
-	<div>
+	<div class="danger-zone">
```

Comment:
![image](https://github.com/user-attachments/assets/9c80979a-2257-4f64-8fc8-e525ee746aa9)
Total margin top and bottom should be the opposite around notecard ?

Reply:
I'd even put it into a separate section.

Leave/Delete conversation is destructive operations that, as said, cannot be undone. Archive is just a UI filter.

Reply:
Then I'll move it to Personal section